core/src/test/java/org/frankframework/pipes/IsUserInRolePipeTest.java

```diff
@@ -0,0 +1,304 @@
+		when(securityHandler.isUserInRole(any(String.class))).thenReturn(true);
+
+		setNotInRoleForward();
+		pipe.configure();
+
+		// When
+		PipeRunResult prr = doPipe(pipe, ROLE, mockedSession);
+
+		// Expect
+		assertEquals(PipeForward.SUCCESS_FORWARD_NAME, prr.getPipeForward().getName());
+	}
+
+	@Test
+	public void userIsNotInSingleRoleAttribute() throws Exception {
+		// Given
+		ISecurityHandler securityHandler = mock(ISecurityHandler.class);
+		PipeLineSession mockedSession = mock(PipeLineSession.class);
+		when(mockedSession.getSecurityHandler()).thenReturn(securityHandler);
+		when(securityHandler.isUserInRole(any(String.class))).thenReturn(false);
+
+		setNotInRoleForward();
+		pipe.setRole(ROLE);
+		pipe.configure();
+
+		// When
+		PipeRunResult prr = doPipe(pipe, "", mockedSession);
+
+		// Expect
+		assertEquals(NOT_IN_ROLE_FORWARD_NAME, prr.getPipeForward().getName());
+	}
+
+	@Test
+	public void userIsNotInSingleRoleInput() throws Exception {
+		// Given
+		ISecurityHandler securityHandler = mock(ISecurityHandler.class);
+		PipeLineSession mockedSession = mock(PipeLineSession.class);
+		when(mockedSession.getSecurityHandler()).thenReturn(securityHandler);
+		when(securityHandler.isUserInRole(any(String.class))).thenReturn(false);
+
+		setNotInRoleForward();
+		pipe.configure();
+
+		// When
+		PipeRunResult prr = doPipe(pipe, ROLE, mockedSession);
+
+		// Expect
+		assertEquals(NOT_IN_ROLE_FORWARD_NAME, prr.getPipeForward().getName());
+	}
+
+	@Test
+	public void userIsInMultiRoleInput() throws Exception {
+		// Given
+		ISecurityHandler securityHandler = mock(ISecurityHandler.class);
+		PipeLineSession mockedSession = mock(PipeLineSession.class);
+		when(mockedSession.getSecurityHandler()).thenReturn(securityHandler);
+
+		when(securityHandler.isUserInRole(ROLE)).thenReturn(false);
+		when(securityHandler.isUserInRole(ROLE2)).thenReturn(true);
+		when(securityHandler.isUserInRole(ROLE3)).thenReturn(true);
+
+		setNotInRoleForward();
+		pipe.configure();
+
+		// When
+		PipeRunResult prr = doPipe(pipe, ROLE+","+ROLE2+","+ROLE3, mockedSession);
+
+		// Expect
+		assertEquals(PipeForward.SUCCESS_FORWARD_NAME, prr.getPipeForward().getName());
+	}
+
+	@Test
+	public void userIsInMultiRoleInputWithForwards() throws Exception {
+		// Given
+		ISecurityHandler securityHandler = mock(ISecurityHandler.class);
+		PipeLineSession mockedSession = mock(PipeLineSession.class);
+		when(mockedSession.getSecurityHandler()).thenReturn(securityHandler);
+
+		when(securityHandler.isUserInRole(ROLE)).thenReturn(false);
+		when(securityHandler.isUserInRole(ROLE2)).thenReturn(true);
+		when(securityHandler.isUserInRole(ROLE3)).thenReturn(true);
+
+		setNotInRoleForward();
+		pipe.registerForward(new PipeForward(ROLE, ""));
+		pipe.registerForward(new PipeForward(ROLE2, ""));
+		pipe.registerForward(new PipeForward(ROLE3, ""));
+		pipe.configure();
+
+		// When
+		PipeRunResult prr = doPipe(pipe, ROLE+","+ROLE2+","+ROLE3, mockedSession);
```

Comment:
Perhaps create a new private var for the combination, since you're using this construct 3 times as well. `ROLE+","+ROLE2+","+ROLE3`